backend/layers/processing/utils/dask_utils.py

```diff
@@ -0,0 +1,22 @@
+import logging
+
+import tiledb
+from scipy import sparse
+
+logger = logging.getLogger(__name__)
+
+
+class TileDBSparseArrayWriteWrapper:
```

Comment:
Nice abstraction

Reply:
i'm glad you think so, you wrote it LOL

Reply:
LOL, let me just pat myself on the back.